README.md

```diff
@@ -31,6 +31,7 @@ document_store = QdrantDocumentStore(
     url="localhost",
     index="Document",
     embedding_dim=512,
+    hnsw_config={"m": 16, "ef_construct": 64},
```

Comment:
Could you please make the parameter optional and pass `None` as the default value? I want to make sure the client defaults are always used.

Reply:
It was made optional, has a default `None` value and clarification in the README.